### ISSUE TITLE
Be able to add a new recipient on the fly

### ISSUE THREAD

**Issue by qubist** [lbraun/thanksgiving#54]
It's hard to start a new donation and then have to stop and add a new recipient and then go back and start the donation again.

**Comment by qubist** [comment]
I'm down to work on this if I can get this set up for development on my computer :)

**Comment by qubist** [comment]
Maybe the flow of this is just if you click "add recipient" from the create donation page, it redirects you back into that page once you're done with the recipient you just created selected in the recipient dropdown.

**Comment by lbraun** [comment]
Yes please, this sounds awesome. Let's get you set up. Send me a calendar invite for an hour time block?

**Comment by qubist** [comment]
Type-ahead:

It's a text input. As you type, a menu of all options filtered by what you're typing appears below it. And an "add [what you typed]" option is also at the top, which opens a modal.

If you open the modal, and enter a new recipient info, and then click "add & select" it populates this field with the new recipient
